index.js

```diff
@@ -37,6 +37,11 @@ var main = function () {
     abbr: 'c',
     help: 'directory where dependencies will be cached'
   });
+  parser.option('deflate', {
```

Comment:
Is there any way to get this working without providing a value?

EG. my command line is:
`npm-cache install --cacheDirectory c:\npm-cache --deflate npm --production`

But running that gives error:
```
'--deflate' expects a value


Usage: C:\Program Files\nodejs\node.exe index.js install [options]

Options:
   -c, --cacheDirectory   directory where dependencies will be cached  [D:\Users\dev_user\.package_cache]
   -v, --version          displays version info and exit
   -r, --forceRefresh     force installing dependencies from package manager without cache  [false]
```

So I have to run it as
`npm-cache install --cacheDirectory c:\npm-cache --deflate y npm --production`

Which seems a little odd.

Also the help file text doesn't list the new flag :smile: